.github/workflows/homebrew-core-head.yml

```diff
@@ -1,14 +1,14 @@
 # Verify Homebrew Core Formula Works
-name: homebrew-core-head
+name: Homebrew Nightly Verification
 
 on:
+  workflow_dispatch:
   schedule:
-    # * is a special character in YAML so you have to quote this string
-    - cron: "0 12 * * *"
+    - cron: "0 0 * * *"
```

Comment:
```suggestion
    - cron: "26 9 * * *"
```

I usually put a random number to spread the load for GHA's hosted runners — they're probably at their slowest at midnight